common/src/main/java/com/wynntils/core/overlays/OverlayManager.java

```diff
@@ -22,33 +21,16 @@ public class OverlayManager {
     private static final List<OverlayHolder> OVERLAYS = new ArrayList<>();
 
     public static void registerOverlay(Object overlayObject) {
-        OverlayHolder overlay = overlayFromObject(overlayObject);
-    }
-
-    private static OverlayHolder overlayFromObject(Object overlayObject) {
         Class<?> overlayClass = overlayObject.getClass();
 
-        List<OverlayOptionHolder> options = new ArrayList<>();
+        for (Field field : FieldUtils.getFieldsWithAnnotation(overlayClass, Overlay.class)) {
+            if (field.getType() != OverlayPosition.class)
+                continue; // For an overlay to exist it must have a position, it not being one means it isn't valid
```

Comment:
Log something here as that's probably unwanted

Reply:
I have it in the commit I'm working on.